src/makefile.amiga

```diff
@@ -6,6 +6,7 @@
 #                             \___|\___/|_| \_\_____|
 #
 # Copyright (C) 1998 - 2020, Daniel Stenberg, <daniel@haxx.se>, et al.
+# SPDX-License-Identifier: curl
```

Comment:
I think inserting the SPDX-lines at the bottom of the header is the least intrusive way.

Reply:
Personally I dislike the license notices a bit as they really are duplicated information that eats up a lot of space and sometimes leads to ambiguities (not here as the curl license is somewhat unique).

But of course the decision is your's and from REUSE's perspective this is totally fine.